Source/SourceKittenFramework/String+SourceKitten.swift

```diff
@@ -80,11 +80,17 @@ extension NSString {
     */
     public func byteRangeToNSRange(start start: Int, length: Int) -> NSRange? {
         let string = self as String
-        return string.indexOfByteOffset(start).flatMap { stringStart in
-            return string.indexOfByteOffset(start + length).map { stringEnd in
-                return NSRange(location: stringStart, length: stringEnd - stringStart)
-            }
+        let startUTF8Index = string.utf8.startIndex.advancedBy(start)
+        let endUTF8Index = startUTF8Index.advancedBy(length)
+        
+        guard let startUTF16Index = startUTF8Index.samePositionIn(string.utf16),
```

Comment:
At first since I filed https://github.com/jpsim/SourceKitten/issues/119, I tested caching full results of offsets. But that was not efficient.
By https://github.com/realm/SwiftLint/pull/263, times of calling `byteRangeToNSRange()` is omitted  on bottle neck case, so I did not include that.